packages/utils/src/report/get-css-code-snippet.ts

```diff
@@ -0,0 +1,111 @@
+import { ChildNode } from 'postcss';
+
+const getCodeSnippetPrefix = (node: ChildNode): string => {
+    if (node.type === 'rule') {
+        return `${node.selector} {`;
+    }
+
+    if (node.type === 'atrule') {
+        return `@${node.name} ${node.params} {`;
+    }
+
+    return '';
+};
+
+const getCodeSnippetContent = (node: ChildNode, content: string): string => {
+    let numberOfSpaces = 4;
+    const grandpa = node.parent ? node.parent.parent : null;
+
+    if (grandpa && grandpa.type === 'atrule') {
+        numberOfSpaces = 8;
+    }
+
+    const spaces = new Array(numberOfSpaces + 1).join(' ');
+
+    return `${spaces}${content}`;
+};
+
+const getCodeSnippetPostfix = (node: ChildNode): string => {
+    let numberOfSpaces = 0;
+    const grandpa = node.parent ? node.parent.parent : null;
+
+    if (grandpa) {
+        numberOfSpaces = 4;
+    }
+
+    const spaces = new Array(numberOfSpaces + 1).join(' ');
+
+    return `${spaces}}`;
```

Comment:
Again, requires deleting `const spaces = ...` above.

```suggestion
    return '}'.padStart(numberOfSpaces, ' ');
```